### PR TITLE
Hide beacon operation field in log if it's 0

### PR DESCRIPTION
**Pull request by terencechain**
**What type of PR is this?**

> Feature

**What does this PR do? Why is it needed?**

Hide displaying rest of the beacon fields if the count is 0

```
INFO blockchain: Finished applying state transition attestations=44 attesterSlashings=0 deposits=0 proposerSlashings=0 voluntaryExits=0
```
```
INFO blockchain: Finished applying state transition attestations=44 slot=3573
```
**Which issues(s) does this PR fix?**

Fixes # N/A

**Other notes for review**
Tried run time and it seems fine. Although non-attestations weren't involved...

### CODE REVIEW

File: beacon-chain/blockchain/log_test.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+		b    *ethpb.BeaconBlock
+		want string
+	}{
+		{name: "empty block body",
+			b:    &ethpb.BeaconBlock{Body: &ethpb.BeaconBlockBody{}},
+			want: "slot=0",
+		},
+		{name: "has attestation",
+			b:    &ethpb.BeaconBlock{Body: &ethpb.BeaconBlockBody{Attestations: []*ethpb.Attestation{{}}}},
+			want: "attestations=1",
```
Comment (mohamedmansour) [inline beacon-chain/blockchain/log_test.go:23]: This only tests if the string contains attestations, maybe we should add an additional test to make sure the other ones are not there.

**Review by mohamedmansour — APPROVED** [review]

**Comment by potuz** [comment]
This breaks scripts that rely on the fields existing. I don't really like to have the same log message have different log structure. 

 If it's just a cosmetic change for the way it's printed then I think this should be applied only when the logger is stdout. 

Notice that if you log to journald, you can request only nonzero fields to be printed

**Comment by terencechain** [comment]
Updating target to v1.2.0

**Comment by rauljordan** [comment]
Hey @terencechain whats the status on this PR?

**Comment by terencechain** [comment]
Ready. Let's target this for 1.3.0

**Review by rauljordan — APPROVED** [review]

**Pull request merged**